packages/core/http/core-http-server-internal/src/http_config.ts

```diff
@@ -205,10 +205,10 @@ const configSchema = schema.object(
         },
       }
     ),
-    // allow access to internal routes by default to prevent breaking changes in current offerings
+    // disable access to internal routes by default
     restrictInternalApis: offeringBasedSchema({
```

Comment:
nit: since we are defaulting to `true` in both offferings could we simplify to

```suggestion
    restrictInternalApis: schema.boolean({ defaultValue: true })
```